tardis/visualization/widgets/shell_info.py

```diff
@@ -224,302 +191,201 @@ def __init__(self, hdf_fpath):
+        self.shells_title = pn.pane.Markdown("### Shells", margin=(0, 0, 5, 0), styles={'font-size': '14px', 'font-weight': 'bold', 'color': '#333'})
+        self.shells_table = pn.widgets.Tabulator(
+            shells_df,
+            layout="fit_data",
+            selectable=1,
+            widths={"Shell No.": 80, "Rad. Temp.": 120, "Dilution Factor": 120},
+            titles={"Shell No.": "Shell No.", "Rad. Temp.": "Rad. Temp. (K)", "Dilution Factor": "W"},
+            styles={'border': '1px solid #ddd', 'font-size': '12px', 'background-color': '#fff'},
+            text_align="center",
+            header_align="center",
         )
 
-        # Creating the element count table widget
-        self.element_count_table = create_table_widget(
-            self.data.element_count(self.shells_table.df.index[0]),
-            [15, 30, 55],
-            changeable_col={
-                "index": -1,  # since last column will change names
-                # Shells table index will give all possible shell numbers
-                "other_names": [
-                    f"Frac. Ab. (Shell {shell_num})"
-                    for shell_num in self.shells_table.df.index
-                ],
-            },
+        # Element count table (Shell 1 initially)
+        element_df = self.data.element_count(1)
+        self.element_title = pn.pane.Markdown("### Elements (Shell 1)", margin=(0, 0, 5, 0), styles={'font-size': '14px', 'font-weight': 'bold', 'color': '#333'})
+        self.element_count_table = pn.widgets.Tabulator(
+            element_df,
+            layout="fit_data",
+            selectable=1,
+            widths={"Element": 100, "Frac. Ab. (Shell 1)": 140},
+            styles={'border': '1px solid #ddd', 'font-size': '12px', 'background-color': '#fff'},
+            text_align="center",
+            header_align="center",
         )
 
-        # Creating the ion count table widget
-        self.ion_count_table = create_table_widget(
-            self.data.ion_count(
-                self.element_count_table.df.index[0],
-                self.shells_table.df.index[0],
-            ),
-            [20, 30, 50],
-            changeable_col={
-                "index": -1,
-                # Since element are same for each shell thus previous table
-                # (element counts for shell 1) will give all possible elements
-                "other_names": [
-                    f"Frac. Ab. (Z={atomic_num})"
-                    for atomic_num in self.element_count_table.df.index
-                ],
-            },
+        # Ion count table (initially empty, but ensure data is populated on selection)
+        self.ion_title = pn.pane.Markdown("### Ions (No Selection)", margin=(0, 0, 5, 0), styles={'font-size': '14px', 'font-weight': 'bold', 'color': '#333'})
+        self.ion_count_table = pn.widgets.Tabulator(
+            pd.DataFrame(columns=["Species", "Frac. Ab."]),
+            layout="fit_data",
+            selectable=1,
```

Comment:
Given that a lot of these calls to Tabulator have the same arguments, it may be reasonable to wrap this into a method with most of these arguments pre-set.